dpgen/generator/arginfo.py

```diff
@@ -321,12 +321,14 @@ def fp_style_siesta_args() -> List[Argument]:
     doc_kspacing = 'Sample factor in Brillouin zones.'
     doc_mixingweight = 'Proportion a of output Density Matrix to be used for the input Density Matrix of next SCF cycle (linear mixing).'
     doc_NumberPulay = 'Controls the Pulay convergence accelerator.'
+    doc_fp_pp_path = 'Directory of psuedo-potential or numerical orbital files to be used for 02.fp exists.'
+    doc_fp_pp_files = 'Psuedo-potential file to be used for 02.fp. Note that the order of elements should correspond to the order in type_map.'
 
     args = [
         Argument("ecut", int, optional=False, doc=doc_ecut),
         Argument("ediff", float, optional=False, doc=doc_ediff),
-        Argument("kspacing", float, optional=False, doc=doc_kspacing),
-        Argument("mixingweight", float, optional=False, doc=doc_mixingweight),
+        Argument("kspacing", [float, int], optional=False, doc=doc_kspacing),
```

Comment:
Why `kspacing` can be `int`?

Reply:
It's not clear to me if the following example is a typo or not. I am not familiar with siesta.

https://github.com/deepmodeling/dpgen/blob/8dea29ef125f66be9641afe5ac4970433a9c9ce1/examples/run/deprecated/dp0.12-lammps-siesta/dp-lammps-siesta/CH4/param_CH4.json#L129

Reply:
cc @Silvia-liu

Reply:
It should be `"kspacing" : 1,`

Reply:
It should be `"kspacing" : 1.0,`